flexport/latitude
# Upgrade react-popper and react-popper-tooltip

This PR upgrades `react-popper` and `react-popper-tooltip` to their latest versions. It turns out we're only a few patch versions behind head

react-popper: `1.3.4` => `1.3.7` https://github.com/popperjs/react-popper/releases (performance fixes)
react-popper-tooltip: `2.10.0` => `2.10.1` no changelog

Manual Testing Plan:
Checkout this branch, ensure that stories including popups work as expected (SearchableSelectInput, Popover, etc).
